src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -688,6 +740,7 @@
   "fluid.tconstruct.molten_netherite": "熔融下界合金",
   "fluid.tconstruct.flowing_molten_netherite": "流动的熔融下界合金",
   "item.tconstruct.molten_netherite_bucket": "熔融下界合金桶",
+  "fluid.tconstruct.molten_netherite.fluid_effect": "熔融下界合金有最高的基础伤害，迫不及待看着你的目标流下鲜血。",
```

```suggestion
  "fluid.tconstruct.molten_netherite.fluid_effect": "熔融下界合金有最高的基础伤害，适合在等不及让你的目标流下鲜血时使用。",
```
Not a good suggestion tho, not required.

---

```suggestion
  "fluid.tconstruct.molten_netherite.fluid_effect": "熔融下界合金有最高的基础伤害，足以让你的目标血流如注。",
```